static/js/ui.js

```diff
@@ -164,7 +164,7 @@ exports.maybe_show_deprecation_notice = function (key) {
  */
 exports.switchToFullWidth = function () {
     $("#full-width-style").remove();
-    $('head').append('<style id="full-width-style" type="text/css">' +
+    $('head').append('<style id="full-width-style">' +
```

Comment:
Can you rebase this commit on to your branch (and it may already be on master before you see this comment)?:

https://github.com/zulip/zulip/pull/9900/commits/bdc219d69de4ff157962b1c4bda7cf1980f02de8

The first commit of #9900 basically removes this code, which has been dead code for many years.

Reply:
Done.